src/functions/avl-processor/index.ts

```diff
@@ -132,11 +191,12 @@ export const handler: SQSHandler = async (event, context) => {
     const {
         AVL_SUBSCRIPTION_TABLE_NAME: avlSubscriptionTableName,
         AVL_VALIDATION_ERROR_TABLE_NAME: avlValidationErrorTableName,
+        GTFS_TRIP_MAPS_TABLE_NAME,
```

Comment:
Should we override the variable name here to be gtfsTripMapsTableName to be consistent with the others? Or vice versa remove the overrides